Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -784,30 +793,22 @@ namespace xr
                 ActionResources.CONTROLLER_GET_THUMBSTICK_CLICK_ACTION_LOCALIZED_NAME,
                 ActionResources.CONTROLLER_GET_THUMBSTICK_CLICK_PATH_SUFFIX,
                 &ActionResources.ControllerGetThumbstickClickAction,
-                bindings,
+                microsoftControllerBindings,
                 instance);
 
+            // Provide default suggested bindings to instance
+            XrInteractionProfileSuggestedBinding suggestedBindings{ XR_TYPE_INTERACTION_PROFILE_SUGGESTED_BINDING };
+            XrCheck(xrStringToPath(instance, ActionResources.DEFAULT_XR_INTERACTION_PROFILE, &suggestedBindings.interactionProfile));
+            suggestedBindings.suggestedBindings = defaultBindings.data();
+            suggestedBindings.countSuggestedBindings = (uint32_t)defaultBindings.size();
+            XrCheck(xrSuggestInteractionProfileBindings(instance, &suggestedBindings));
+
             // Provide Microsoft suggested binding to instance
             XrInteractionProfileSuggestedBinding microsoftSuggestedBindings{ XR_TYPE_INTERACTION_PROFILE_SUGGESTED_BINDING };
             XrCheck(xrStringToPath(instance, ActionResources.MICROSOFT_XR_INTERACTION_PROFILE, &microsoftSuggestedBindings.interactionProfile));
-            microsoftSuggestedBindings.suggestedBindings = bindings.data();
-            microsoftSuggestedBindings.countSuggestedBindings = (uint32_t)bindings.size();
-
-            // Fallback on the default bindings if the Microsoft bindings fail
-            if (XR_FAILED(xrSuggestInteractionProfileBindings(instance, &microsoftSuggestedBindings)))
-            {
-                ControllerInfo.ControllerBinding = false;
```

Comment:
you also might want to remove the "ControllerBinding" parameter from ControllerInfo all together.